### PR TITLE
fix: pagesize limit bug fix and password check fix

### PR DESCRIPTION
**Pull request by o0OooO**
## Description

Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### CODE REVIEW

**Comment by NingLu** [comment]
We identified new issues on unchanged lines of code. Navigate to the Amazon CodeGuru Reviewer console to view the recommendations to fix them.

File: aws_extension/cloud_api_manager/api_manager.py [hunk 1]
```diff
@@ -215,7 +215,8 @@ def upsert_role(self, role_name, permissions, creator, user_token=""):
     def upsert_user(self, username, password, roles, creator, initial=False, user_token=""):
```
Comment (NingLu) [inline aws_extension/cloud_api_manager/api_manager.py:215]: Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Possible hardcoded password: &#39;&#39; https://bandit.readthedocs.io/en/latest/plugins/b107_hardcoded_password_default.html

File: aws_extension/cloud_api_manager/api_manager.py [hunk 2]
```diff
@@ -215,7 +215,8 @@
...
         if not self.auth_manger.enableAuth and not initial:
             return {}
-
+        if not password or len(password) < 1:
+            raise Exception('password should not be none')
```
Comment (NingLu) [inline aws_extension/cloud_api_manager/api_manager.py:219]: Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Using `Exception` and `BaseException` can make your code prone to errors and difficult to maintain. Instead, we recommend using one of the [Built-in Exceptions](https://docs.python.org/3/library/exceptions.html) or creating a custom exception class that is derived from `Exception` or one of its subclasses.



Similar issue at line number 238.

**Review by alvindaiyan — APPROVED** [review]
yes, but need clean not used code

**Pull request merged**